quicdoq_cli_app/quicdoq_app.c

```diff
@@ -87,6 +87,9 @@
 
 #endif
 
+#include "picoquic_binlog.h"
+#include "picoquic_logger.h"
+
```

Comment:
Yes of course.

The log function reference were pulled out of the main header file in picoquic to allow application to only link with the logging functions that they need, and so manage the size of the executable.